pkg/apis/pingcap/v1alpha1/defaulting/tidbcluster.go

```diff
@@ -70,6 +71,9 @@ func setTidbClusterSpecDefault(tc *v1alpha1.TidbCluster) {
 	if tc.Spec.PVReclaimPolicy == nil {
 		tc.Spec.PVReclaimPolicy = &retainPVP
 	}
+	if len(tc.Spec.StatefulSetUpdateStrategy) == 0 {
+		tc.Spec.StatefulSetUpdateStrategy = apps.RollingUpdateStatefulSetStrategyType
+	}
```

Comment:
We cannot default it here, it's possible that users configure this field for each component but not this field, just return the default value in `func (a *componentAccessorImpl) StatefulSetUpdateStrategy()`.

Reply:
That's right, I'll fix it